Classes/New Issue/NewIssueTableViewController.swift

```diff
@@ -99,6 +99,11 @@ final class NewIssueTableViewController: UITableViewController, UITextFieldDeleg
         
         // Make the return button move on to description field
         titleField.delegate = self
+        titleField.addTarget(
+            self,
+            action: #selector(NewIssueTableViewController.textFieldEditingChanged),
```

Comment:
Are we not able to use the titleField delegate we already define for this functionality? Transversely, do we need to remove the target in deinit?

Reply:
Checked text field delegate method `shouldChangeCharactersInRange` and thought it would need extra care when handling characters since the method has to return a `bool` for changing the text. 

Forgot about the remove target. Thx for pointing that out